docs/integrations/azure/discovery.md

```diff
@@ -62,7 +67,7 @@ The parameters available are:
 
 - **`host:`** _(optional)_ Leave empty for Cloud hosted, otherwise set to your self-hosted instance host.
 - **`organization:`** Your Organization slug (or Collection for on-premise users). Required.
-- **`project:`** Your project slug. Required.
+- **`project:`** Your project slug. Required. Wildcards are supported as show on the examples above. If not set, all projects will be searched.
```

Comment:
No longer required right?

Reply:
@MarcBruins I think the suggestion here is to make this 

**`project:`** Your project slug. _Optional_. Wildcards are supported as show on the examples above. If not set, all projects will be searched.